test/models/user_test.rb

```diff
@@ -0,0 +1,57 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+  	@user = users(:one)
+  end	
+  
+  test 'user has many friends' do
+    assert 1 < @user.friends.count 
+  end
+
+  test 'user has many posts' do
```

Comment:
Please also run me through on how you created posts for your test users

Reply:
Posts fixtures